near-mock-vm/vm/tests/utils.js

```diff
@@ -0,0 +1,76 @@
+
```

Comment:
Is there a way to reuse `near-mock-vm/vm/src/utils.ts` instead of copy-pasting?

Reply:
Yeah this was an old artifact.